libraries/thingy_52/app_error_weak.h

```diff
@@ -40,6 +40,8 @@
 #ifndef APP_ERROR_WEAK_H__
 #define APP_ERROR_WEAK_H__
 
+#include <stdint.h>

```

Comment:
Why do we need that?
If there is no need for it then please remove this commit.

Reply:
The order of the includes of the header files is changed by formatting the files. Reordering the includes of the headers leads to an error since this file uses `uint32_t` but didn't include `stdint.h`